README.md

```diff
@@ -22,8 +22,20 @@ For vanilla and wrapper layouts scripts and styles must still be manually requir
 - Provides `NODE_ENV` to templates via `__environment`
 - `__isProduction` is `true` if `NODE_ENV` equals `PRODUCTION`
 - Provides a range of [handlebars helpers](#handlebars-helpers), including template inheritance
+- instruments `fetch` to send data about server-to-server requests to graphite. By default capi1, capi2, sapi and elastic search are instrumented. To add more services pass in a 'serviceDependencies' option (see examples below)
 
 
+var serviceMatchers = {
+	'capi-v1-article': /^https?:\/\/api\.ft\.com\/content\/items\/v1\/[\w\-]+/,
+	'capi-v1-page': /^https?:\/\/api\.ft\.com\/site\/v1\/pages\/[\w\-]+/,
+	'capi-v1-pages-list': /^https?:\/\/api\.ft\.com\/site\/v1\/pages/,
+	'sapi': /^https?:\/\/api\.ft\.com\/content\/search\/v1/,
+	'elastic-mget': /^https?:\/\/[\w\-]+\.foundcluster\.com:9243\/v1_api_v2\/item/,
```

Comment:
Why? Because it's a temporary solution?

I see no problem with putting everything we use in 2 or more apps in here. Removing them from the list wouldn't be a breaking change. It's actually quite nice because if we intend to remove the service globally we can delete the listing here and any lingering use of the service will start sending messages to sentry.